gossipd/routing.c

```diff
@@ -733,27 +833,34 @@ bool handle_pending_cannouncement(struct routing_state *rstate,
 	if (pending->updates[1])
 		handle_channel_update(rstate, pending->updates[1]);
 
+	process_pending_node_announcement(rstate, &pending->node_id_1);
+	process_pending_node_announcement(rstate, &pending->node_id_2);
+
 	tal_free(pending);
 	return local && forward;
 }
 
 /* Return true if this is an update to a pending announcement (and queue it) */
 static bool update_to_pending(struct routing_state *rstate,
 			      const struct short_channel_id *scid,
-			      const u8 *update, const u8 direction)
+			      u32 timestamp, const u8 *update,
```

Comment:
Would `timestamp` be better as a `time_t`?

Reply:
In this case it's from the update message, where it's a u32, so it's OK.

Reply:
Yep, trying to be consistent with the underlying type in the message.